homeassistant/components/heos/strings.json

```
@@ -31,6 +31,28 @@
       "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]"
     }
   },
+  "options": {
+    "step": {
+      "init": {
+        "title": "HEOS Options",
+        "description": "You can sign-in to your HEOS Account to access favorites, streaming services, and other features. Clearing the credentials will sign-out of your account.",
+        "data": {
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        },
+        "data_description": {
+          "username": "The username or e-mail address of your HEOS Account.",
```

```suggestion
          "username": "The username or email address of your HEOS Account.",
```
think is the spelling that is normall yused